flow/connectors/postgres/qvalue_convert.go

```diff
@@ -19,8 +19,8 @@ import (
 	"github.com/PeerDB-io/peerdb/flow/shared"
 )
 
-func (c *PostgresConnector) postgresOIDToName(recvOID uint32) string {
-	if ty, ok := pgtype.NewMap().TypeForOID(recvOID); ok {
+func (c *PostgresConnector) postgresOIDToName(typeMap *pgtype.Map, recvOID uint32) string {
```

Comment:
Don't need to pass typeMap parameters around when they're on the method receiver